cves/kernel/CVE-2020-15393.yml

```diff
@@ -452,7 +452,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: I believe this vulnerability was the result of a lapse by the programmer. Freeing memory you allocated before is standard procedure in C, but even an experienced programmer can forget to do so. Based on what I saw, they did not implement protection against further leaks.
```

Comment:
readability would greatly improve with splitting lines here. but good answer! i like that you've used the terminologies specified.